Common/Data/Custom/SmartInsider/SmartInsiderTransaction.cs

```diff
@@ -357,5 +357,25 @@ public override string ToLine()
                 PriceAdjustedFactor,
                 TreasuryHolding);
         }
+
+        /// <summary>
+        /// Attempts to normalize and parse SmartInsider transaction dates.
+        /// </summary>
+        /// <param name="date">Date string to parse</param>
+        /// <returns>DateTime object</returns>
+        /// <exception cref="ArgumentException">Date string was unable to be parsed</exception>
+        public DateTime ParseTransactionDate(string date)
```

Comment:
nit - maybe this guy could be static

Seems `SmartInsiderIntention` is performing similar operations, think we should address it there too https://github.com/QuantConnect/Lean/blob/master/Common/Data/Custom/SmartInsider/SmartInsiderIntention.cs#L157, maybe move this method into the `SmartInsiderEvent`